packages/pierce-dom/.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "../../babel.config.json"
```

Comment:
shouldn't the storybook generator be configured to add this to every new package?